server/src/main/java/org/opensearch/action/search/CreatePITAction.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.search;
+
+import org.opensearch.action.ActionType;
+
```

Comment:
We're working toward re-enabling javadoc checks. Can you please add class level javadocs?